lib/lumberjack/server.rb

```diff
@@ -214,13 +226,14 @@ def initialize(fd)
       super()
       @parser = Parser.new
       @fd = fd
+      @closed = Concurrent::AtomicBoolean.new
```

Comment:
Should the `closed` value here be shared among the server and each connection? Otherwise, what instructs the active connections to close?